test/test_exaerror.lua

```diff
@@ -1,97 +1,93 @@
+package.path = "src/?.lua;" .. package.path
+
 local luaunit = require("luaunit")
-local exaerror = require("exaerror")
+local exaerror = require("ExaError")
```

Comment:
```suggestion
local ExaError = require("ExaError")
```
-> also rename in the rest of the file